### PR TITLE
New feature : Mobile - Get Weather

### PR DESCRIPTION
**Pull request by widhofaisal**

### CODE REVIEW

File: controller/user/explore_monitoring_controller.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+	if err_bind := c.Bind(&coordinate); err_bind != nil {
+		log.Print(color.RedString(err_bind.Error()))
+		return c.JSON(http.StatusBadRequest, map[string]interface{}{
+			"status":  400,
+			"message": "bad request",
+		})
+	}
+
+	apikey := "869a5f0aa562d21ec64ff37c7c6c157f"
+	baseURL := "https://api.openweathermap.org/data/2.5/weather"
```
Comment (dvnggak) [inline controller/user/explore_monitoring_controller.go:31]: LGTM !! 🥇

File: controller/user/explore_monitoring_controller.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	var label string
+
+	weather, ok := data["weather"].([]interface{})
+	if ok && len(weather) > 0 {
+		weatherData, ok := weather[0].(map[string]interface{})
+		if ok {
+			labelValue, ok := weatherData["main"].(string)
+			if ok {
+				switch labelValue {
+				case "Thunderstorm", "Drizzle", "Rain", "Snow":
```
Comment (dvnggak) [inline controller/user/explore_monitoring_controller.go:94]: Nice Work on Switch case for defining weather labels 💯

**Review by dvnggak — APPROVED** [review]
LGTM 👍🏻

**Comment by widhofaisal** [comment]
cuman 1 endpoint codingnya sampe 160 line :")

**Pull request merged**